### DISCUSSION TITLE
Please add working examples

### DISCUSSION THREAD

**Discussion by thepwrtank18** [ookii-dialogs/ookii-dialogs-winforms#18] [Q&A] [answered]
I'm trying to figure out how the Task Dialog works, but I don't know how to create a button since the example just assumes you do, and nothing online uses WinForms.

**Comment by augustoproiete** [comment]
@thepwrtank18 There are examples of each dialog (Task Dialog included) in the repo: https://github.com/ookii-dialogs/ookii-dialogs-winforms/tree/master/sample

What's missing?

**Comment by thepwrtank18** [comment]
What does `_customTaskDialogButton` mean? It's not defined anywhere.

**Comment by augustoproiete** [comment, marked as answer]
All dialogs are Windows Forms controls that you are meant to drag to your form, and use the Windows Forms Visual Editor to configure (though you can do everything manually if you prefer).

![image](https://user-images.githubusercontent.com/177608/110514211-742a7780-80dd-11eb-8657-087cdd3ee56f.png)

![image](https://user-images.githubusercontent.com/177608/110514122-5ceb8a00-80dd-11eb-996f-0b5da0bbf6a5.png)

![image](https://user-images.githubusercontent.com/177608/110514159-6543c500-80dd-11eb-9f3e-2fcedf48cdfc.png)

**Comment by thepwrtank18** [comment]
Thanks for that. I'll close the issue now.
